introducing-signal-store.md

```diff
@@ -0,0 +1,701 @@
+* Simple refactor: If you used MiniRx Store before, refactor to Signal Store will be straight-forward: change the TypeScript imports, remove the Angular async pipes (and ugly non-null assertions (`!`)) from the template
+
+### Getting Started
+To install the @mini-rx/signal-store package, use your package manager of choice:
+
+`npm install @mini-rx/signal-store`
+
+### Use-cases
+
+MiniRx Signal Store is highly flexible and offers three different well-defined state containers out of the box:
+
+- Store (Redux)
+- Feature Store
+- Component Store
+
+All three can be easily used together in your application.
+Depending on the use-case, you can choose the state container which suits your needs.
+
+These are the typical use-cases:
+
+![Image description](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/qj5anf3ibwrjw20yu6t0.png)
+
+## Store (Redux)
+Let's start with Redux... If you do not like Redux, do not run away! MiniRx Signal Store has first-class support for bypassing the infamous Redux boilerplate: Feature Stores.
+
+### Redux Pattern
+
+The Redux pattern is great to manage state at large scale. MiniRx Signal Store offers a powerful Redux API.
+
+#### The Redux ingredients
+Let's have a look at the basic building blocks of the Redux pattern:
+
+- Actions: objects which describe events with an optional payload
+- Reducers
+  - pure functions which know how to update state (based on the current state and a given action)
+  - reducers are run for every action in order to calculate the next state
+- Effects: listen to a specific action, run side effects like API calls and handle race conditions (with RxJS flattening operators)
+- Memoized selectors: pure functions which describe how to select state from the global state object
+- Store
+  - holds the global state object
+  - wires everything up (reducers, effects)
+  - exposes the public Store API (`dispatch`, `select`)
+
+You can see already, that these building blocks offer a nice separation of concerns.
+Every part does its own specialized thing, you can also split up your code more easily:
+
+![Image description](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/ib6mnr0vmrq550muhys7.png)
+
+#### More benefits of Redux
+- Single source of truth for your state (one global state object)
+- Uni-directional data flow
+- Indirection of State and Actions
+  - State: state is updated via actions, state changes are propagated via new Signal values
+  - Actions: A single action can update state in one or many reducers. The same action can trigger async tasks in effects.
+- Tooling: Redux DevTools
+
+### Redux Store Implementation
+
+The MiniRx Signal Store makes use of both Signals and Observables. Let's see how the Redux Store is implemented...
+
+#### 🚦 Signals
+
+- The global state is implemented as Angular Signal, because Signals are great for state!
```

Comment:
> because Signals are great for state
maybe that's subjectively my thing 😅 but saying something XY is because something else is "great" is not very precise 😉 how about sth like "because signals provide the best balance between simplicity and robustness for app state"

Reply:
yes not precise, will fix it.